walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/requirements/VerifiedIdRequirement.kt

```diff
@@ -39,6 +41,21 @@ class VerifiedIdRequirement(
 
     internal var verifiedId: VerifiedId? = null
 ): Requirement {
+    // Constraint that represents how the requirement is fulfilled
+    internal var constraint: VerifiedIdConstraint = toVcTypeConstraint()
+
+    internal fun toVcTypeConstraint(): VerifiedIdConstraint {
```

Comment:
In order to make Constraints classes internal, constraints couldn't be used as a parameter in the constructor of VerifiedIdRequirement as that class is public. So moved it inside the class as a field and since type cannot be empty and it is mandatory, initialized with it here. CredentialPresentationInputDescriptor.toConstraint() creates other constraints like claims, accepted issuers and if merges them with type to create a Groupconstraint.